php/class-sync.php

```diff
@@ -388,6 +392,10 @@ public function is_syncable( $attachment_id ) {
 			$syncable = true;
 		}
 
+		if ( $this->managers['media']->is_oversize_media( $attachment_id ) ) {
```

Comment:
@pereirinha - Perhaps add this at the end of the chain above, or add `true === $syncable` in front, to stop hitting the file system on files that are not syncable anyway.